metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -257,12 +261,53 @@ class Compilers(
       Nil
     }
 
+  def completions(
+      path: AbsolutePath,
+      breakpointPosition: LspPosition,
+      token: CancelToken,
+      expression: CompletionsArguments
+  ): Future[Seq[DebugCompletionItem]] = {
+
+    val compiler = loadCompiler(path).getOrElse(fallbackCompiler)
+
+    val input = path.toInputFromBuffers(buffers)
+    val metaPos = breakpointPosition.toMeta(input)
+    val oldText = metaPos.input.text
+    val lineEnd = oldText.indexOf("\n", metaPos.start)
+    val semicolonSuffix = "; "
+    val modified =
+      oldText.substring(0, lineEnd) + semicolonSuffix + expression
+        .getText() + oldText
+        .substring(lineEnd)
```

Comment:
Maybe this would be a little bit easier to understand?
```suggestion
    // insert expression at the end of breakpoint's line
    val modified = s"""|${oldText.substring(0, lineEnd)}; ${expression.getText()}
                       |${oldText.substring(lineEnd + 1)}
                       |""".stripMargin
```

Reply:
Using strip margin was a bit complex especially that we would want to keep all the existing spaces. I changed to string interpolation.